src/jttt/Core/Game.java

```diff
@@ -17,16 +17,16 @@
     private int gameType;
     private HashMap<Mark, Player> players;
 
-    public Game(PlayerFactory playerFactory) {
+    public Game(Board board, PlayerFactory playerFactory) {
```

Comment:
Can the two constructors be refactored such that duplication between them can be removed?